backend/lib/services/members/MemberManager.js

```diff
@@ -97,6 +97,21 @@ class MemberManager {
     return this.subjectList.members
   }
 
+  async deleteSecret (id) {
+    const item = this.subjectList.get(id)
+    if (!item) {
+      return this.subjectList.members
```

Comment:
We do not modify the member list. I would not return a body?

Reply:
You are right. I think I just did it as we always did in the past. I will have another look.